### ISSUE TITLE
Profiler crash

### ISSUE THREAD

**Issue by akrasnogor-ms** [defold/defold#7059]
**Describe the bug (REQUIRED)**
Our autotest team got a crash in Profiler, while perform some autotests. We don't have any sentry issue and dump yet.
But we have partial log. Our QA will try to reproduce that crash and get some additional information.

We have some line with error in log before crash was happened:
`10-12 15:57:16.129 27395 27533 E defold  : ERROR:DLIB: REMOTERY: DEFOLD: rmtHashTable_Insert: The hash was 0x89abcdef. Setting it to 0x89abcdef ('<empty>')`

And crash itself:

```
10-12 16:02:14.316 27395 27395 W Thread-197712: type=1400 audit(0.0:902960): avc: denied { ioctl } for path="socket:[73523103]" dev="sockfs" ino=73523103 ioctlcmd=0x8927 scontext=u:r:untrusted_app:s0:c29,c257,c512,c768 tcontext=u:r:untrusted_app:s0:c29,c257,c512,c768 tclass=udp_socket permissive=0
--------- beginning of crash
10-12 16:02:15.879 27395 27520 F libc    : ../src/remotery/lib/Remotery.c:4246: void rmtMessageQueue_CommitMessage(Message *, MessageID): assertion "LoadAcquire((rmtU32*)&message->id) == MsgID_NotReady" failed
10-12 16:02:16.156  3231  3231 W linker  : Warning: "/data/app/com.google.android.webview-9M7H-CwrgbjKbbSvyqgqOg==/base.apk!/lib/arm64-v8a/libcrashpad_handler_trampoline.so" unused DT entry: unknown processor-specific (type 0x70000001 arg 0x0) (ignoring)
10-12 16:02:16.225  3231  3231 W linker  : Warning: "/data/app/com.google.android.webview-9M7H-CwrgbjKbbSvyqgqOg==/base.apk!/lib/arm64-v8a/libmonochrome.so" unused DT entry: unknown processor-specific (type 0x70000001 arg 0x0) (ignoring)
10-12 16:02:16.330   880  1519 D AF::Track: interceptBuffer: took 297us to intercept 0 tracks
10-12 16:02:16.367   885  1654 W DisplayIdentification: Invalid EDID: falling back to serial number due to missing display name.
10-12 16:02:16.367   885  1654 W DisplayIdentification: Invalid EDID: falling back to ASCII text due to missing serial number.
10-12 16:02:16.368   801  1000 I SDM     : HWCDisplay::GetColorModeCount: Supported color mode count = 1
10-12 16:02:16.368   801  1000 I SDM     : HWCDisplay::GetColorModes: Color mode = 0 is supported
10-12 16:02:16.856  3231  3231 W linker64: type=1400 audit(0.0:902961): avc: denied { read } for name="u:object_r:mm_video_prop:s0" dev="tmpfs" ino=13927 scontext=u:r:untrusted_app:s0:c29,c257,c512,c768 tcontext=u:object_r:mm_video_prop:s0 tclass=file permissive=0
10-12 16:02:16.871  3231  3231 E libc    : Access denied finding property "persist.vendor.debug.en.drpcrpt"
10-12 16:02:17.048  3231  3231 W chromium: [1012/160217.046181:WARNING:thread_snapshot_linux.cc(112)] Unknown scheduling policy 1073741825
10-12 16:02:17.210 27395 27520 I defold  : INFO:CRASH: Successfully wrote Crashdump to file: /data/user/0/com.MelsoftGames.FamilyIslandFarm/files/_crash
10-12 16:02:17.210 27395 27520 E defold  : ERROR:CRASH: CALL STACK:
10-12 16:02:17.210 27395 27520 E defold  : 
10-12 16:02:17.210 27395 27520 E defold  : # 0 pc        0x638 [vdso] <unknown>+0
10-12 16:02:17.210 27395 27520 E defold  : # 1 pc      0x420ec ...oid.runtime/lib64/bionic/libc.so <unknown>+0
10-12 16:02:17.210 27395 27520 E defold  : 
10-12 16:02:17.210 27395 27520 E defold  : 
10-12 16:02:17.632  1859  2891 I ActivityManager: Process com.google.android.webview:sandboxed_process0:org.chromium.content.app.SandboxedProcessService0:0 (pid 27858) has died: fore BTOP
10-12 16:02:17.632  1859  1995 I libprocessgroup: Successfully killed process cgroup uid 99440 pid 27858 in 0ms
```

**To Reproduce (REQUIRED)**
No specific steps.

**Defold version (REQUIRED):**
  - Version  1.3.4 (2d3564f13cf2b2ff15bcd6d258d6aae6c103d049)

**Platforms (REQUIRED):**
The crashes were observed on the:
Redmi Note 9 Pro (Android 10)
oppo reno 2 CPH1907 (android 11)
SM-A715F (android 11)
Xiaomi POCO X3 NFC M2007J20CG (android 10)
Huawei P30 Pro VOG-L29 (Android 10)
D53gAP, iPhone 12, iphoneos, arm64e, 15.1.1, 19B81

**Logs (OPTIONAL):**
I attached log.
[crash_log.txt](https://github.com/defold/defold/files/9766736/crash_log.txt)

**Comment by akrasnogor-ms** [comment]
Crashdump 
[_crash.zip](https://github.com/defold/defold/files/9777185/_crash.zip)

**Comment by AGulev** [comment]
it looks like the same issue : https://github.com/defold/defold/issues/6989

**Comment by AGulev** [comment]
Fixed in 1.3.7
